src/screens/BitcoinNetwork/BitcoinNetworkIntro.js

```diff
@@ -0,0 +1,250 @@
+*/
+import * as React from 'react';
+import styled from 'styled-components/native';
+import { FlatList, Platform } from 'react-native';
+import { CachedImage } from 'react-native-cached-image';
+import { connect } from 'react-redux';
+import type { NavigationScreenProp } from 'react-navigation';
+
+import { BLOCKCHAIN_NETWORK_TYPES } from 'constants/blockchainNetworkConstants';
+import ContainerWithHeader from 'components/Layout/ContainerWithHeader';
+import { ScrollWrapper } from 'components/Layout';
+import { BoldText, MediumText } from 'components/Typography';
+import Icon from 'components/Icon';
+import Button from 'components/Button';
+import SlideModal from 'components/Modals/SlideModal';
+import CheckPin from 'components/CheckPin';
+import { LabelBadge } from 'components/LabelBadge';
+
+import { baseColors, fontStyles } from 'utils/variables';
+import { responsiveSize } from 'utils/ui';
+
+import { resetIncorrectPasswordAction } from 'actions/authActions';
+import { setActiveBlockchainNetworkAction } from 'actions/blockchainNetworkActions';
+import { initializeBitcoinWalletAction } from 'actions/bitcoinActions';
+import { ASSETS } from 'constants/navigationConstants';
+
+import type { EthereumWallet } from 'models/Wallet';
+import type { Dispatch } from 'reducers/rootReducer';
+
+type Props = {
+  navigation: NavigationScreenProp<*>,
+  initializeBitcoinWallet: (wallet: EthereumWallet) => void,
+  resetIncorrectPassword: () => void,
+  setActiveBlockchainNetwork: (id: string) => void,
+};
+
+type State = {
+  showPinScreenForAction: boolean,
+  processingCreate: boolean,
+};
+
+const CustomWrapper = styled.View`
+  flex: 1;
+  padding: 20px 55px 40px 46px;
+`;
+
+const Title = styled(BoldText)`
+  color: ${baseColors.pomegranate};
+  ${fontStyles.rJumbo};
+`;
+
+const BodyText = styled(MediumText)`
+  color: ${baseColors.pomegranate};
+  ${fontStyles.rBig};
+  margin-top: ${responsiveSize(26)}px;
+`;
+
+const ListItemWrapper = styled.View`
+  flex-direction: row;
+  justify-content: flex-start;
+  margin-top: ${responsiveSize(19)}px;
+`;
+
+const ContentWrapper = styled.View`
+  align-items: stretch;
+  margin-left: ${responsiveSize(19)}px;
+  flex: 1;
+  flex-wrap: wrap;
+`;
+
+const Label = styled(MediumText)`
+  color: ${baseColors.pomegranate};
+  ${fontStyles.rLarge};
+`;
+
+const Subtext = styled(MediumText)`
+  color: ${baseColors.pomegranate};
+  ${fontStyles.rBig};
+  margin-top: ${responsiveSize(10)}px;
+`;
+
+const FeatureIcon = styled(CachedImage)`
+  height: 124px;
+  width: 124px;
+  margin-bottom: 24px;
+`;
+
+const Wrapper = styled.View`
+  position: relative;
+  margin: 5px 20px 20px;
+  padding-top: ${Platform.select({
+    ios: '20px',
+    android: '14px',
+  })};
+`;
+
+const ButtonWrapper = styled(Wrapper)`
+  padding: 0 46px 20px;
+`;
+
+const features = [
+  {
+    key: 'btc',
+    label: 'Bitcoin.',
+    subtext: 'No waiting for miner or block confirmations. Seriously instant transactions.',
```

Comment:
This text is incorrect, we don't have ppn for bitcoin

Reply:
You're right, I've used the PPN view as a template, but we don't know what's gonna be here for now, so it's been removed :+1: